sickbeard/manual_snatch.py

```diff
@@ -125,26 +125,27 @@ def collectEpisodesFromSearchThread(show):
     # Queued Searches
     searchstatus = SEARCH_STATUS_QUEUED
     for searchThread in sickbeard.searchQueueScheduler.action.get_all_ep_from_queue(show):
-        episodes += getEpisodes(searchThread, searchstatus)
+        if isinstance(searchThread, sickbeard.search_queue.ManualSearchQueueItem):
```

Comment:
why are you checking this? get_all_ep_from_queue() only returns ManualSearchQueueItem. So now your validating redundant?